.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,52 @@
+name: benchmark
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  pytest-benchmark:
+    name: pytest-benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout pymovements repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
```

Comment:
while I think people *should* use the newest version of python, it is very unrealisitic. even bigger libraries have not adopted to 3.13 yet, so maybe benchmarking on <3.13, e.g. 3.10 or so would be more practical.

Reply:
This is the definition of a GitHub workflow. Github workflows are internal to github and neither targeted at the user nor at the developer. They are handled purely by the CI.

we should benchmark on the newest python versions with the newest optimizations.
I don't care much about performance on old python systems.